### ISSUE TITLE
Glyphicons should recommend the use of aria-hidden=true

### ISSUE THREAD

**Issue by kkirsche** [twbs/bootstrap#16493]
I think that it would be good to recommend that Glyphicons, the icon font, should use `aria-hidden="true"`. I believe that this should help users who may use third party icon fonts. For example, certain icon fonts if you wrote 

```html
<style>
  .icon-star:before { content: "★ "; }
</style>

<span><span class="icon-star" aria-hidden="true"></span>Favorite</span>
```

Normally, this would be read as "Black Star Favorite". Use of `aria-hidden="true"` would cause VoiceOver to announce "Favorites" instead, which I believe is more accurate for most cases.

Thoughts?

cc: @patrickhlauke

**Comment by patrickhlauke** [comment]
We already do that...so you mean actually explicitly saying that in the docs?

**Comment by patrickhlauke** [comment]
Oh and we do already have a boxout for accessibility right after the glyphicon section that says this, so unless I'm missing something....closing?
